pyaedt/modules/DesignXPloration.py

```diff
@@ -83,9 +83,11 @@ class CommonOptimetrics(object):
     
     name :
     
-    dictinputs :
+    dictinputs 
+    
+    optimtype : str
+        Type of the optimization.
     
-    optimtype :
     """
     @property
```

Comment:
```suggestion

    @property
```
Always have line break between class docstring and subsequent methods.